loom/script/serialize/lsBinReader.cpp

```diff
@@ -784,6 +782,9 @@ Assembly *BinReader::loadExecutable(LSLuaState *_vm, utByteArray *byteArray)
     {
         Reference *ref = lmNew(NULL) Reference;
         ref->name     = readPoolString();
+
+        lmAssert(ref->name[0] != 0, "Reference name is empty, try recompiling the executable");
```

Comment:
Can you clarify if this is script binary or native binary? ie should I run lsc or gcc?